### ISSUE TITLE
Error when trying to add a ref to a styled component

### ISSUE THREAD

**Issue by rubenmoya** [modulz/stitches#110]
Hello!

I'm migrating a small project from emotion/styled-system to stitches and I'm having some problems with the `ref` attribute.

I've created [a codesandbox](https://codesandbox.io/s/stitchesstyled-ref-rv0m9?file=/src/App.tsx) if you want to have a look at the error, this is the code:

```tsx
import * as React from "react";
import { createStyled } from "@stitches/styled";

const { styled } = createStyled({});

const Box = styled("div", {});

export default function App() {
  const boxRef = React.useRef<HTMLDivElement>(null);
  const divRef = React.useRef<HTMLDivElement>(null);

  return (
    <div>
      <Box ref={boxRef}>Hello there</Box>
      <div ref={divRef}>General Kenobi</div>
    </div>
  );
}
```

The error seems to be `Property 'ref' does not exist on type [...]` (although the error text is huge), am I missing something? 🤔

Besides that problem I'm loving it so far :) much more cleaner than using emotion/styled-system 👏 I can't wait for the official release

**Comment by peduarte** [comment]
Hey @rubenmoya thanks for raising this issue ✌️ We'll take a look.

Let me know how you get on with the migration!

**Comment by rubenmoya** [comment]
Hello! Sorry to reopen this one, but typescript still complains about the `ref` attribute.

The error is:

```
Property 'ref' does not exist on type 'IntrinsicAttributes &...
```

I'm using the `9.0.0-alpha.4` version, you can check it out in [this codesandbox](https://codesandbox.io/s/stitchesstyled-ref-rv0m9?file=/src/App.tsx)

**Comment by hadihallak** [comment]
@rubenmoya sorry about that. the fix above was for the library itself not forwarding refs internally and not the type, however, this issue along with a few other type issues have already been resolved and are scheduled for a release on Monday 😄

**Comment by hadihallak** [comment]
Fixed in #134
